app/models/user.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  PASSWORD_COMPLEXITY_REGEXP = /(?=.*?\w)(?=.*?[0-9])(?=.*?[#?!@$%^&*-])/.freeze
+
+  devise :database_authenticatable, :rememberable, :validatable, :recoverable, :confirmable
+
+  validates :first_name, :last_name, presence: true
+  validate :password_complexity
+
+  enum role: {
```

Comment:
Same here. `:role` is no longer present on the user. https://github.com/Wolfpack-Digital/rails-template/pull/20#discussion_r826301452